Expensify/App
# Update app with new branding

### Details
This updates the app with our new logo and colors.

### Fixed Issues
N/A

### Tests / QA Steps
1. Go to the Expensify.cash sign in page and make sure the screenshot, logo, and colors are updated on all platforms
2. On web, make sure the favicon is also updated to the new logo
3. Make sure the app icon is updated to use the new logo too.

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/2319350/116123657-20b5be00-a678-11eb-8e5a-f51f2af72d1c.png)

![image](https://user-images.githubusercontent.com/2319350/116123249-a2591c00-a677-11eb-88da-7128fb9e5d03.png)

![image](https://user-images.githubusercontent.com/2319350/116123464-dfbda980-a677-11eb-9b44-65728fc7d10f.png)

![image](https://user-images.githubusercontent.com/2319350/116123730-388d4200-a678-11eb-895e-1a6bc5a133d9.png)


#### Mobile Web
![image](https://user-images.githubusercontent.com/2319350/116124597-57d89f00-a679-11eb-9080-cc8d61cf31a0.png)



#### Desktop
![image](https://user-images.githubusercontent.com/2319350/116125477-64112c00-a67a-11eb-8297-89305668b11e.png)

![image](https://user-images.githubusercontent.com/2319350/116125502-6bd0d080-a67a-11eb-946a-f70c2d9ee50d.png)


#### iOS
![image](https://user-images.githubusercontent.com/2319350/116125158-0250c200-a67a-11eb-810a-17588919fe44.png)


![image](https://user-images.githubusercontent.com/2319350/116124474-2f50a500-a679-11eb-8467-04c3fb7eccce.png)


![image](https://user-images.githubusercontent.com/2319350/116124160-c9fcb400-a678-11eb-9dd7-9d9fd18991f2.png)


![image](https://user-images.githubusercontent.com/2319350/116124160-c9fcb400-a678-11eb-9dd7-9d9fd18991f2.png)


#### Android
